### PR TITLE
Dry up kind property in session api

### PR DESCRIPTION
**Pull request by ralphiee22**
### Summary

Creates roles array in one shot. Now whenever we add a kind to roles, it will automatically appear in the session response, if the user has the new role.

### Reviewer guidance

Does order matter in terms of the kind array?
ex. (['admin', 'coach'] as opposed to ['coach', 'admin'])

### References

Cleans up issue discussed here
https://trello.com/c/C2jQqcUQ/96-dry-up-the-kind-property-in-sessionserializer

----

### Contributor Checklist

- [x] Contributor has fully tested the PR manually
- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [N/A] If there are any front-end changes, before/after screenshots are included
- [N/A] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

**Comment by jonboiser** [comment]
@ralphiee22 This is the only part of the code that picks a particular item off of `session.kind`

https://github.com/learningequality/kolibri/blob/d52caa3c7b8e15feefa01d72c0537973994c823c/kolibri/plugins/facility_management/assets/src/views/user-page/edit-user-modal.vue#L276

There is also UI code that kind of enforces a business rule that a user only have one facility-level role at at a time:
https://github.com/learningequality/kolibri/blob/d52caa3c7b8e15feefa01d72c0537973994c823c/kolibri/plugins/facility_management/assets/src/state/actions/rolesActions.js#L17

So, in theory `session.kind` could have more than one item, but the working assumption for awhile is that it _only has one item_

File: kolibri/auth/api.py [hunk 1]
```diff
@@ -366,23 +366,17 @@ def get_session(self, request):
                    'full_name': user.full_name,
                    'user_id': user.id,
                    'can_manage_content': user.can_manage_content}
-        roles = Role.objects.filter(user_id=user.id)
+
+        roles = list(Role.objects.filter(user_id=user.id).values_list('kind', flat=True).distinct())
```
Comment (jonboiser) [inline kolibri/auth/api.py:370]: if you are in the `if isinstance(user, AnonymousUser) === False` bloc, can you safely assume that `user.roles` is defined?

Comment (jonboiser) [reply kolibri/auth/api.py:370]: oh yeah, and i think you're right that with Coach roles scoped to the Classroom + Facility, a user could have more than one role, contradicting what I was saying below.

Comment (ralphiee22) [reply kolibri/auth/api.py:370]: the query will return an empty array if there are no roles defined for the user

**Pull request merged**